lexicon.txt

```diff
@@ -0,0 +1,1317 @@
+aaaa
+aaaaaaa
+aal
+abuflen
+ack
+acked
+ackno
+acks
+acount
+acs
+addr
+addressalignedbeats
+addrstart
+addtogroup
+advertize
+afe
+ahb
+amba
+ap
+api
+apis
+april
+arg
+arp
+arppacket
+arpprocesspacket
+arprefreshcacheentry
+arps
+ascii
+asf
+atmel
+automaticpadcrcstrip
+autonegotiation
+aws
+backoff
+backofflimit
+backpressure
+bactive
+bafter
+barry
+baseaddress
+bbbbbbb
+bbefore
+bcloseaftersend
+bcloserequested
+bconnpassed
+bconnprepared
+bds
+berkeley
+besr
+bexpired
+bfinaccepted
+bfinacked
+bfinlast
+bfinrecv
+bfinsent
+bhasinit
+bisforrx
+bist
+bitmask
+bitscanreverse
+bl
+blowwater
+bmallocerror
+bmayconnect
+bmsschange
+bool
+boolean
+boutstanding
+bpassaccept
+bpassqueued
+bpdg
+breleaseaftersend
+breusesocket
+broadcastframesreception
+brxstopped
+bsd
+bsendfullsize
+bsendkeepalive
+btimestamps
+buf
+buffermanagement
```

Comment:
check to see if this should be two words in the text

Reply:
Thanks. But this cannot be split into two words since this is one of the directories in the portable section of the TCP repository and it is included in a lot of comments.